packages/bbui/src/Form/Core/Multiselect.svelte

```diff
@@ -24,6 +24,7 @@
   export let loading: boolean = false
   export let onOptionMouseenter = () => {}
   export let onOptionMouseleave = () => {}
+  export let searchPlaceholder: string | undefined = undefined
```

Comment:
NAB: I think you might be able to do

```
export let searchPlaceholder: string = "Search"
```

And that way you don't need the `??` below.

EDIT: Yes you did that in Picker.svelte in the file below 🙂